assignments/prototypes.js

```diff
@@ -15,13 +16,28 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
+function GameObject(GrandChildAttributes){
```

Comment:
GrandChildAttributes could be a confusing parameter, since technically this is the parent (with Characterstats being the child and Humanoid being the grandchild)